checker/src/abstract_value.rs

```diff
@@ -1480,6 +1502,7 @@ impl AbstractValueTrait for Rc<AbstractValue> {
         if let Some(result) = self
             .get_cached_interval()
             .greater_or_equal(&other.get_cached_interval())
+            //TODO: add comparison for octagons
```

Comment:
You would do that as a separate statement following this one. While the cost/benefit impact is still unclear, it may wise to add an option that controls this.